### PR TITLE
feat: multi transaction

### PR DESCRIPTION
**Pull request by alaibe**
New endpoint to send many transaction in one request

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status_go.html))?
- [ ] Have you tested changes with mobile?
- [ ] Have you tested changes with desktop?

File: appdatabase/migrations/sql/1657617291_add_multi_transactions_table.up.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS multi_transactions (
+    from_address VARCHAR NOT NULL,
+    from_asset VARCHAR NOT NULL,
+    from_amount VARCHAR NOT NULL,
+    to_address VARCHAR NOT NULL,
+    to_asset VARCHAR NOT NULL,
+    type VARCHAR NOT NULL,
```
Comment (richard-ramos) [inline appdatabase/migrations/sql/1657617291_add_multi_transactions_table.up.sql:7]: Would it be a good idea to also store the decimals?

File: services/wallet/transaction.go [hunk 2]
```diff
@@ -31,18 +65,19 @@ const (
...
+	From               common.Address `json:"from"`
+	To                 common.Address `json:"to"`
+	Data               string         `json:"data"`
+	Symbol             string         `json:"symbol"`
+	GasPrice           bigint.BigInt  `json:"gasPrice"`
+	GasLimit           bigint.BigInt  `json:"gasLimit"`
+	Type               PendingTrxType `json:"type"`
+	AdditionalData     string         `json:"additionalData"`
+	ChainID            uint64         `json:"network_id"`
+	MultiTransactionId int64          `json:"multi_transaction_id"`
```
Comment (richard-ramos) [inline services/wallet/transaction.go]:
```suggestion
	ChainID            uint64         `json:"networkId"`
	MultiTransactionId int64          `json:"multiTransactionId"`
```

Also, probably `MultiTransactionId` should be named `MultiTransactionID`. I don't know if the linter catches this

File: services/wallet/transaction.go [hunk 3]
```diff
@@ -179,6 +214,89 @@ func (tm *TransactionManager) watch(ctx context.Context, transactionHash common.
...
+	}
+
+	insert, err := tm.db.Prepare(`INSERT OR REPLACE INTO multi_transactions
+                                      (from_address, from_asset, from_amount, to_address, to_asset, type, timestamp)
+                                      VALUES
+                                      (?, ?, ?, ?, ?, ?, ?)`)
+	if err != nil {
+		return nil, err
+	}
+	result, err := insert.Exec(
```
Comment (richard-ramos) [inline services/wallet/transaction.go:230]:
```suggestion
	defer insert.Close()

	result, err := insert.Exec(
```

**Review by richard-ramos — APPROVED** [review]
Nice! just left some minor comments!

File: appdatabase/migrations/sql/1657617291_add_multi_transactions_table.up.sql (hunk 1 above)
Comment (alaibe) [reply appdatabase/migrations/sql/1657617291_add_multi_transactions_table.up.sql:7]: :thinking: you mean from and to decimals?

**Review by Khushboo-dev-cpp — APPROVED** [review]

File: appdatabase/migrations/sql/1657617291_add_multi_transactions_table.up.sql (hunk 1 above)
Comment (alaibe) [reply appdatabase/migrations/sql/1657617291_add_multi_transactions_table.up.sql:7]: we store the asset symbol which are our key, we have the decimals in the store already

**Pull request merged**